test/client/rs.test.js

```diff
@@ -0,0 +1,9 @@
+describe('The RS implementation', function() {
+  it('encodes HELLO as URYY\\. (ROT13 example)', function() {
+    assert.equal(RS.encode('HELLO'), 'URYY\\');
+  });
+
+  it('decodes URYYB as HELLO. (ROT13 example)', function() {
+    assert.equal(RS.decode('URYY\\'), 'HELLO');
```

Comment:
whoops, thank you! this is kinda sketchy/hasty ROT which doesn't wrap around at the end of the alphabet like it should, just keeps going and uses whatever ASCII character is at the computed ord even if it's out of alphabetical range. *technically* should be URYYB but in my sketchy version it's URYY\. will fix :)